server/protocols/jmap/src/main/java/org/apache/james/jmap/model/MailboxFactory.java

```diff
@@ -111,9 +112,16 @@ private Mailbox fromMessageManager(MessageManager messageManager, Optional<List<
             .totalMessages(mailboxCounters.getCount())
             .sortOrder(SortOrder.getSortOrder(role))
             .sharedWith(Rights.fromACL(metaData.getACL()))
+            .namespace(getNamespace(mailboxPath.getUser(), mailboxSession))
             .build();
     }
 
+    @VisibleForTesting MailboxNamespace getNamespace(String mailboxPathUser, MailboxSession mailboxSession) {
```

Comment:
s/mailboxPathUser/mailboxPath/g IMO And the argument should be a mailboxPath.

That might allow to handle mailboxPath.getNamespace in the future